lib/telephony/pinpoint/voice_sender.rb

```diff
@@ -55,14 +57,15 @@ def send(message:, to:)
       def client_configs
         @client_configs ||= Telephony.config.pinpoint.voice_configs.map do |voice_config|
           credentials = AwsCredentialBuilder.new(voice_config).call
-          args = { region: voice_config.region, retry_limit: 0 }
-          args[:credentials] = credentials unless credentials.nil?
```

Comment:
Ditto about the eager `.map { AwsCredentialBuilder.new.call }` I think we should find a way to make this lazier